tox.ini

```diff
@@ -5,15 +5,15 @@ envlist = black,
           flake8,
           docstrings,
           py{36,37,38,39,launch},
-          func-s_{base,sklearn,metaflow,tf{115,21,25,26},service,docs,
+          func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,ray112,ray2,service,docs,
                   imports{1,2,3,4,5,6,7,8,9,10,11},noml,grpc}-py37,
           standalone-{cpu,gpu,tpu,local}-py38,
           func-cover,
           cover
 
 [base]
 setenv =
-    YEA_WANDB_VERSION = 0.8.6
+    YEA_WANDB_VERSION = 0.8.7
```

Comment:
This will have to be 0.8.8, we released it yesterday for the unmock-server.

Reply:
(also, pinning flask<2.2.2 is not necessary, we have a hack that overcomes it -- see my comment to your yea-wandb PR)

Reply:
reverted